components/atoms/SVGIcon/svg-icon.tsx

```diff
@@ -11,8 +11,8 @@ interface IconProps {
 
 const SVGIcon: React.FC<IconProps> = ({ onClick, className, IconImage, size = 16 }) => {
   return (
-    <div className={clsx(`inline-flex`, className)} onClick={onClick}>
-      <svg>
+    <button onClick={onClick}>
+      <svg className={clsx(`inline-flex`, className)}>
```

Comment:
Refactor (blocking): The CSS classes should be on the button as they were on the div previously.

```suggestion
    <button className={clsx(`inline-flex`, className)} onClick={onClick}>
      <svg>
```